tests/test_dataclasses.py

```diff
@@ -102,7 +102,11 @@ def test_dataclass_properties_and_class_variables() -> None:
         class Point:
             x: float
             y: float
-            z: ClassVar[float] = 3
+
+            # These definitions create class variables
+            r: ClassVar[float]
+            s: float = 3
```

Comment:
So the test still passes but `s` should appear in the `__init__` method parameters right?

Reply:
Yeah right!

I suspected an edge case because attribute annotation expression is sliced and the `ClassVar` is removed. It was the more obvious solution. That means the solution should check for `instance-attribute` as-well.